jupyterlab_gcpscheduler/src/service/gcp.ts

```diff
@@ -295,33 +330,70 @@ export class GcpService {
     pageToken?: string
   ): Promise<Schedules> {
     try {
-      const filter = `labels.job_type=${SCHEDULED_JOB_INDICATOR}`;
       const projectId = await this.projectId;
-      const params: { [k: string]: string } = { filter };
+      const params: { [k: string]: string } = {};
       if (pageSize) {
         params['pageSize'] = String(pageSize);
       }
       if (pageToken) {
         params['pageToken'] = pageToken;
       }
       const response = await this._transportService.submit<
-        ListAiPlatformJobsResponse
+        ListSchedulesResponse
       >({
-        path: `${AI_PLATFORM}/projects/${projectId}/jobs`,
+        path: `${NOTEBOOKS_API_BASE}/projects/${projectId}/locations/-/schedules`,
         params,
       });
+      const schedules = [];
+      if (
+        response.result &&
+        response.result.unreachable &&
+        !response.result.schedules
+      ) {
+        throw {
+          result: {
+            error: {
+              status: 'UNREACHABLE',
+              message: response.result.unreachable.join(', '),
+            },
+          },
+        };
+      }
+      if (response.result && response.result.schedules) {
+        const promises: Promise<Execution>[] = [];
+        for (const notebooksApiSchedule of response.result.schedules) {
+          promises.push(
+            this.getLatestExecutionForSchedule(notebooksApiSchedule.displayName)
```

Comment:
We want to catch a promise failure for an execution request so that it won't prevent all of the other schedules from rendering.

Reply:
I catch that in the getLatestExecutionForSchedule method as it's easier to handle there and no other method uses it